opba-db/src/main/resources/migration/migrations/0001-init-protocol-facade.xml

```diff
@@ -384,4 +384,28 @@
                                  referencedColumnNames="bank_uuid"
                                  referencedTableName="${table-prefix}bank_profile"/>
     </changeSet>
+    <changeSet id="bank_validation_rule_table" author="mhr">
+        <createTable tableName="${table-prefix}bank_validation_rule">
```

Comment:
Please add column `mode` that has `IGNORE` value - meaning that this is validation ignore rule